LibTest/js_interop/JSExportedDartFunction/jsExportedDartFunction_t01.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A JavaScript callable function created from a Dart function.
+///
+/// @description Check that a native Dart function can be passed to JS and
+/// called from JS.
+/// @author sgrekhov22@gmail.com
+
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+import '../../../Utils/expect.dart';
+import '../js_utils.dart';
+
+@JS()
```

Comment:
Avoid annotating non-externals - they don't do anything ("someone" should really add a warning... :)).